cares_reinforcement_learning/memory/memory_buffer.py

```diff
@@ -40,6 +40,29 @@ def sample(self, batch_size):
         transposed_batch = zip(*experience_batch)
         return transposed_batch
 
+    def sample_next(self, batch_size):
```

Comment:
wonder if if there is a better name than sample_next

Reply:
How about sample_two_steps()?